packages/core/src/civil.ts

```diff
@@ -60,6 +61,10 @@ export class Civil {
     return this.ethApi.toBigNumber(num);
   }
 
+  public async signMessage(message: string): Promise<EthSignedMessage> {
+    return this.ethApi.signMessage(message, this.userAccount);
```

Comment:
Is this the correct method to use for signing the Constitution?

@ritave, I think you're the knowledgable party here